### PR TITLE
Svg logo

### PR DESCRIPTION
**Pull request by sineade**
:art:
Changed logo to an .svg file so that it is scalable and will show up nicely on retina screens or other high resolution screens. The last logo was fuzzy on retina screens. 

https://ama-digital.myjetbrains.com/youtrack/issue/GCDES-1026

screenshot: 
screen shot 2016-03-17 at 10 47 26 am

### CODE REVIEW

File: app/assets/stylesheets/ama_layout/layout_components/sidebar.scss [hunk 1]
```diff
@@ -135,12 +140,19 @@ span.activepage{
   @extend .row;
 }
 
-.title-area li.name img{
-  padding-top: $base-padding*2;
+.title-area {
```
Comment (MathieuGilbert) [inline app/assets/stylesheets/ama_layout/layout_components/sidebar.scss]: I think there shouldn't be spaces before the `{`s.

**Comment by MathieuGilbert** [comment]
:hankey: There are a few spaces before { in the scss file.

Could you also add `*.gem` to the `.gitignore` file? We don't want those files in the repo.

**Comment by MathieuGilbert** [comment]
Could you run:
`git rm --cached ama_layout-2.2.6.gem`
Then commit and push? That will remove the file from the branch. The .gitignore entry only prevents new ones from being added, so since it was already there, it needs to be manually dropped.

**Comment by sineade** [comment]
:wave: 
ran git rm --cached ama_layout-2.2.6.gem

**Comment by MathieuGilbert** [comment]
:art: :art: :art: Nice!

**Pull request merged**